webdriver/tests/support/fixtures_bidi.py

```diff
@@ -58,6 +61,39 @@ async def subscribe_events(events, contexts=None):
             pass
 
 
+@pytest_asyncio.fixture
+async def set_cookie(bidi_session):
+    """
+    Set a cookie and remove them after the test is finished.
+    """
+    cookies = []
+
+    async def set_cookie(cookie, partition=None):
+        partition_descriptor = None
+        set_cookie_result = await bidi_session.storage.set_cookie(cookie=cookie, partition=partition)
+        if set_cookie_result["partitionKey"] != {}:
+            # Make a copy of the partition key, as the original dict is used for assertion.
+            partition_descriptor = copy.deepcopy(set_cookie_result["partitionKey"])
+            partition_descriptor["type"] = "storageKey"
+        # Store the cookie partition to remove the cookie after the test.
+        # The requested partition can be a browsing context, so the returned partition descriptor (it's always of type
+        # "storageKey") is used.
+        cookies.append((copy.deepcopy(cookie), partition_descriptor))
+        return set_cookie_result
+
+    yield set_cookie
+
+    yesterday = datetime.now() - timedelta(1)
+    yesterday_timestamp = time.mktime(yesterday.timetuple())
+
+    for cookie, partition in reversed(cookies):
+        try:
+            cookie["expiry"] = yesterday_timestamp
+            await bidi_session.storage.set_cookie(cookie=cookie, partition=partition)
+        except (InvalidArgumentException, UnableToSetCookieException, UnderspecifiedStoragePartitionException):
```

Comment:
I've just noticed that `UnableToSetCookieException` and `UnderspecifiedStoragePartitionException` are not imported in this file

Reply:
fixed